lib/weather.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'services/current_weather'
```

Comment:
These files do not exist

Reply:
Yeah you are right. I am not to sure why it is not working, it works on my laptop. I think some files are missing, I really don't know why that is to be honest.